config/confighttp/go.mod

```diff
@@ -5,6 +5,7 @@ go 1.20
 require (
 	github.com/golang/snappy v0.0.4
 	github.com/klauspost/compress v1.17.1
+	github.com/peterbourgon/unixtransport v0.0.3
```

Comment:
Do we trust a random 0.0.3 package? Do we have other alternative?

Reply:
We could inline that package into the code here, given that the code itself is very small. It would probably be refactored a bit through that process to make it easier to maintain.